cpp/log/sqlite_db-inl.h

```diff
@@ -21,7 +23,14 @@ DEFINE_bool(sqlite_synchronous_off, false,
 namespace {
 
 
+static cert_trans::Latency<std::chrono::milliseconds, std::string>
+    latency_by_op_ms("database_latency_by_operation_ms", "operation",
```

Comment:
Maybe have "sqlite" somewhere in there, to reflect it's in that file? There might be similar counters for FileDB?